classes/log/store.php

```diff
@@ -28,6 +28,7 @@
 use \tool_log\helper\buffered_writer as helper_writer;
 use \core\event\base as event_base;
 use \stdClass as php_obj;
+use stdClass;
```

Comment:
Is this a duplicate? as use stdClass is already set

Reply:
Not a duplicate the first is aliased as php_obj